docs/client.md

```diff
@@ -24,7 +23,7 @@ eik init
 Fill in the generated `assets.json` file with the necessary details.
 
 For the `server` property, if you are using a locally running asset server
-the server property will likely be `http://localhost:4001`
+the server property will likely be `http://assets.finncdn.no`
```

Comment:
Not sure we should point to our own host for this. Maybe use an example host in a way.

Reply:
sure